bgpd/bgp_ovsdb_if.c

```diff
@@ -2122,10 +2125,6 @@ void bgp_daemon_ovsdb_neighbor_update (struct peer *peer,
         &peer->local_as, 1);
      */
 
-    VLOG_DBG("updating weight to %d\n", peer->weight);
```

Comment:
Taken from #16 , because
1) Neighbor weight is a local-only configuration parameter, there is no reason for BGPD to change it
2) Sometimes BGPD passes incorrect value (out of boundaries I think) and whole transaction gets aborted